subsys/bluetooth/audio/Kconfig.mcs

```diff
@@ -121,4 +121,100 @@ config BT_MCC_SHELL
 	help
 	  This option enables shell support for the Media Control Client.
 
+config BT_MCC_MINIMAL
+	bool "Minimal Media Control Client without optional procedures"
+	default n
+	help
+	  This option disables all optional procedures in the Media Control Client.
+
+config BT_MCC_READ_MEDIA_PLAYER_ICON_URL
+	bool "Support reading Media Player Icon URL"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for reading Media Player Icon URL.
+
+config BT_MCC_READ_TRACK_TITLE
+	bool "Support reading Track Title"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for reading Track Title.
+
+config BT_MCC_READ_TRACK_DURATION
+	bool "Support reading Track Duration"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for reading Track Duration.
+
+config BT_MCC_READ_TRACK_POSITION
+	bool "Support reading Track Position"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for reading Track Position.
+
+config BT_MCC_SET_TRACK_POSITION
+	bool "Support setting Track Position"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for setting Track Position.
+
+config BT_MCC_READ_PLAYBACK_SPEED
+	bool "Support reading Playback Speed"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for reading Playback Speed.
+
+config BT_MCC_SET_PLAYBACK_SPEED
+	bool "Support setting Playback Speed"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for setting Playback Speed.
+
+config BT_MCC_READ_SEEKING_SPEED
+	bool "Support reading Seeking Speed"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for reading Seeking Speed.
+
+config BT_MCC_READ_PLAYING_ORDER
+	bool "Support reading Playing Order"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for reading Playing Order.
+
+config BT_MCC_SET_PLAYING_ORDER
+	bool "Support setting Playing Order"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for setting Playing Order.
+
+config BT_MCC_READ_PLAYING_ORDER_SUPPORTED
+	bool "Support reading Playing Order Supported"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for reading Playing Order Supported.
+
+config BT_MCC_READ_MEDIA_STATE
+	bool "Support reading Media State"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for reading Media State.
+
+config BT_MCC_SET_MEDIA_CONTROL_POINT
+	bool "Support setting Media Control Point"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for setting Media Control Point.
+
+config BT_MCC_READ_MEDIA_CONTROL_POINT_OPCODES_SUPPORTED
+	bool "Support reading Media Control Point Opcodes Supported"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for reading Media Control Point Opcodes Supported.
+
+config BT_MCC_READ_CONTENT_CONTROL_ID
+	bool "Support reading Content Control ID"
+	default !BT_MCC_MINIMAL
+	help
+	  This option enables support for reading Content Control ID.
+
```

Comment:
I cant really mark anything relevant here, but we are missing the following options:
`Read Current Track Segments Object Information
Read Current Track Object Information
Set Current Track Object ID
Read Next Track Object Information
Set Next Track Object ID
Track Discovery – Discover by Current Group Object ID
Set Current Group Object ID
Read Parent Group Object Information`

I understand that these can be controlled through the `CONFIG_BT_OTS_CLIENT` option, however, I can't really see in spec. that they are mandatory IF the OTS Client is enabled. So, I'm politely asking the question whether they are? And if not, should we add options also for these? Possibly grouped together if that makes sense, but dependent on  `CONFIG_BT_OTS_CLIENT`.

Reply:
as to OTS, there is a config to disable all BT_MCC_OTS. Decision has been make if you would like the minimal configuration, the you most likely would disable OTS

Reply:
> Decision has been make if you would like the minimal configuration, the you most likely would disable OTS

I think that's fair for now - It probably would make sense to have the same support for the OTS characteristics as we do for the MCS; i.e. full support to enable/disable the individual ones. You may want to do search (which requires OTS) without any of the ots OTS related ones.

Reply:
I didn't see this one `BT_MCC_OTS` until now :D

I agree, its fair for now